api/acme.go

```diff
@@ -0,0 +1,849 @@
+package api
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/api/acme"
```

Comment:
This should be moved to a different path `certificate/acme` can be a good place.